### PR TITLE
Task build layout

### PR DESCRIPTION
**Pull request by gidspektor**
this is the first task. Created template with nav bar, background image and responsiveness.

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,110 @@
+{box-sizing: border-box}
+
+
+
+body{
+
```
Comment (charliecog) [inline index.css]: add margin: 0; to get rid of white line around everything

File: index.css [hunk 2]
```diff
@@ -0,0 +1,110 @@
+{box-sizing: border-box}
```
Comment (charliecog) [inline index.css:1]: Missing a css selector?
did you mean:
* 
also css should not be all on one line

File: index.css [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+
+
+body{
+
+}
+
+nav{
```
Comment (charliecog) [inline index.css]: need a space between nav and the curly braces (please address this throughout your whole css file)

File: index.css [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+    height: 120px;
```
Comment (charliecog) [inline index.css]: not doing anything

File: index.css [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+    z-index: 3;
+}
+
+nav img{
+    margin-left: 50px;
+}
+
+.navRight{
+    float: right;
+    margin-right: 200px;
```
Comment (charliecog) [inline index.css]: make this a % unit eg: 10% to make it a bit more responsive

File: index.css [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+    margin-top: 70px;
+    display: none;
+    background-image: url("button.png");
+    background-size: 100%;
+    height: 5px;
+    width: 5px;
+    padding: 16px;
+    border: none;
```
Comment (charliecog) [inline index.css]: not doing anything

File: index.css [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+}
+
+nav li{
+    display: inline-block;
+    margin-top: 30px;
+}
+
+.liText{
```
Comment (charliecog) [inline index.css]: not a great class name
think about what it actually is eg: navItem

File: index.css [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+    font-family: "Arial Rounded MT Bold";
```
Comment (charliecog) [inline index.css]: if your going to use this across the whole website just put it as a style on the body. also put a websafe font as a backup

File: index.css (hunk 7 above)
Comment (charliecog) [reply index.css]: actually to be honest you dont need a class. just use a css selector like 
.navRight li a {
//styles
}

File: index.css [hunk 9]
```diff
@@ -0,0 +1,110 @@
...
+    text-transform: uppercase;
+    margin-right: 50px;
+    color: #000000;
+    text-decoration: none;
+    border-bottom: 3px solid transparent;
+    padding: 10px;
+    font-size: 13px;
+}
+
+.liText2{
```
Comment (charliecog) [inline index.css]: needs a better name or try and do without using a specific class

File: index.css [hunk 10]
```diff
@@ -0,0 +1,110 @@
...
+    font-family: "Arial Rounded MT Bold";
```
Comment (charliecog) [inline index.css]: same as before

File: index.css (hunk 9 above)
Comment (charliecog) [reply index.css]: wait is this just a repeat of all the stuff in the previous class? if so then just use 1 class for both and then select 
.dropdown a {
display: none;
}

File: index.html [hunk 11]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="index.css" type="text/css">
+    <title>Gideon Spektor</title>
+</head>
+<body>
+    <nav>
+        <a href="file:///Users/academy/Desktop/portfolio/index.html"><img src="logo.png"></a>
```
Comment (charliecog) [inline index.html]: dont use this link in here, it wont work on anyone elses computer, just replace with a #

Comment (charliecog) [reply index.html]: also you forgot your / at the end of your img self closing tag

File: index.html [hunk 12]
```diff
@@ -0,0 +1,31 @@
...
+        <div class="navRight">
+             <ul>
+                 <li><a href="#about" class="liText">About me</a></li>
```
Comment (charliecog) [inline index.html]: class name needs changing (if you need a class at all)

File: index.html [hunk 13]
```diff
@@ -0,0 +1,31 @@
...
+                 <li><a href="#project" class="liText">Projects</a></li>
+                 <li><a href="#contact" class="liText">Contact</a></li>
+            </ul>
+        </div>
+        <div class="dropdown">
+                <a href="#about" class="liText2">About me</a>
```
Comment (charliecog) [inline index.html]: class naming

**Review by charliecog — COMMENTED** [review]
Please make appropriate changes

File: index.css [hunk 14]
```diff
@@ -0,0 +1,94 @@
+{box-sizing: border-box}
```
Comment (aabdiaziz) [inline index.css:1]: ?

File: index.html [hunk 15]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="index.css" type="text/css">
```
Comment (aabdiaziz) [inline index.html]: might wanna do a different name for the CSS file. something like index.css doesn't describe the file function, where as styles.css would.

File: index.css [hunk 16]
```diff
@@ -0,0 +1,94 @@
...
+}
+
+
+.mainImg {
+    background-image: url("tree.png");
+    background-size: 100% 100%;
+}
+
+
+@media screen and (max-width: 800px) {
```
Comment (aabdiaziz) [inline index.css:84]: Two media screen responsive functions for one size?

**Review by aabdiaziz — COMMENTED** [review]
3 comments to evaluatle

**Pull request merged**